### ISSUE TITLE
Spaces in a sample name (inside the bam file) will cause a failure in the M2 WDL task.

### ISSUE THREAD

**Issue by LeeTL1220** [broadinstitute/gatk#4355]
When you're looking for it, it's easily found:

```
normal_command_line="-I ${normal_bam} -normal `cat normal_name.txt`"
```
If `normal_name.txt` has a space in it, the M2 command is later rendered as:
`.... -normal HCC1143 BL ...` 

This will cause a failure, since "BL" is not a parameter.

We should render as: `.... -normal "HCC1143 BL" ...` 

or `.... -normal HCC1143_BL ...` 

Please note that this affects the tumor sample lines as well.

**Comment by LeeTL1220** [comment]
@davidbenjamin Can this be closed?

**Comment by davidbenjamin** [comment]
Closed by #4360.
